docs/components/gltf-model.md

```diff
@@ -157,6 +159,15 @@ These files are available from the three.js repository, under
 automatically choose whether to use a WASM or JavaScript decoder, so both should
 be included. A Google-hosted version of the Draco decoder libraries saves you from needing to include these libraries in your own project: set `https://www.gstatic.com/draco/v1/decoders/` as the value for `dracoDecoderPath`.
 
+`ktx2TranscoderPath` path must be a folder containing two files:
+
+    basis_transcoder.js — JavaScript wrapper for the WebAssembly transcoder.
+    basis_transcoder.wasm — WebAssembly transcoder.
+
+These files are available from the three.js repository, under
+[examples/js/libs/basis][basis].
```

Comment:
I didn't see `[basis]` defined in the document, similar to `[blender]`. I guess you want to include something like
`[basis]: https://github.com/mrdoob/three.js/tree/dev/examples/js/libs/basis`

Reply:
Good call, bad syntax. Corrected w/ link to that file folder.